src/turbomind/comm/gloo/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+cmake_minimum_required(VERSION 3.8)
+
+include(FetchContent)
+FetchContent_Declare(
+  gloo
+  GIT_REPOSITORY https://github.com/facebookincubator/gloo.git
+  GIT_TAG cbe963b5a43cd75e6eca4f74d2bb38ec8dcfdbc8
+)
+
+# some settings of gloo,
+set(GLOO_INSTALL OFF CACHE BOOL "" FORCE)
+set(GLOO_STATIC_OR_SHARED STATIC CACHE STRING "" FORCE)
+set(__USE_NCCL ${USE_NCCL})
```

Comment:
Unless specified otherwise, CMake values are scoped and no need to backup and recover them.